src/main/java/duke/tasks/Task.java

```diff
@@ -0,0 +1,36 @@
+package duke.tasks;
+
+public class Task {
+    protected String description;
+    protected boolean isDone;
+
+    public String getDescription() {
+        return description;
+    }
+
+    public void setDescription(String description) {
+        this.description = description;
+    }
+
+    public boolean isDone() {
+        return isDone;
+    }
+
+    public void setDone(boolean done) {
```

Comment:
The boolean variable `done` should be named `isDone`. However, this conflicts with your already defined `isDone` within your `Task` class. Could there be a better way to rewrite this?